adapter/googleServiceControl/googleServiceControl_test.go

```diff
@@ -0,0 +1,38 @@
+package googleServiceControl
+
+import (
+	"testing"
+)
+
+func TesthandleMetric(t *testing.T) {
+	request, err := handleMetric()
```

Comment:
follow go unittest convention to https://github.com/golang/go/wiki/TableDrivenTests